src/GovUk.Education.ExploreEducationStatistics.Content.Model/Database/ContentDbContext.cs

```diff
@@ -409,6 +409,27 @@ private static void ConfigureReleaseFile(ModelBuilder modelBuilder)
                         v => v.HasValue
                             ? DateTime.SpecifyKind(v.Value, DateTimeKind.Utc)
                             : null);
+                entity.Property(f => f.PublicApiDataSetVersion)
+                    .HasMaxLength(20)
+                    .HasConversion(
+                        v => v.ToString(),
+                        v => SemVersion.Parse(v, SemVersionStyles.Strict, 20)
+                    );
+
+                entity.HasIndex(rf => new
+                {
+                    rf.ReleaseVersionId,
+                    rf.FileId,
+                })
+                .IsUnique();
```

Comment:
I'm guessing this is unrelated to this PR but we just wanted to add this in because it should have always been there?

Reply:
Yep, this is just to add a unique constraint that should have been there in the first place!